firebase-perf/src/main/java/com/google/firebase/perf/application/FrameMetricsRecorder.java

```diff
@@ -111,6 +111,12 @@ public Optional<PerfFrameMetrics> stop() {
     } catch (IllegalArgumentException err) {
       logger.warn(
           "View not hardware accelerated. Unable to collect FrameMetrics. %s", err.toString());
+      frameMetricsAggregator.reset();
```

Comment:
Do we need `frameMetricsAggregator.reset()` here when it's called after the `catch` blocks?